feeds/grouping-and-source-pinning.md

```diff
@@ -0,0 +1,142 @@
+---
+layout: default
+title: Grouping and Source Pinning
+parent: Feeds
+nav_order: 1
+---
+
+# Grouping and Source Pinning
+
+The default NuGet behavior with feeds declared in NuGet.config is to query each feed for every package. For large repositories, this can lead to performance and network bottlenecks.
+
+By specifying a group for a feed, only the direct and transitive dependencies of a package defined with a `<PackageVersion>` in the same group will use this feed. More information about adding a `CentralManagementGroup` to a `<PackageVersion>` can be found in the [documentation for package version grouping](http://localhost:4000/package-versions/grouping-and-overrides/grouping-and-overrides/).
+
+## Global Group
+
+A global group will be automatically created that includes all `<PackageSource>`, `<PackageVersion>`, and `<PackageReference>` that do not have a `CentralManagementGroup` specified.
+
+`<PackageSource>` that belong to the global group will also be queried for `<PackageReference>` that belong to a `CentralManagementGroup` unless specified by [Feed Scoping](/feeds/grouping-and-source-pinning/#feed-scoping).
+
+## Adding Groups
+
+A feed is added to a group by specifying a `CentralManagementGroup` attribute to the `<PackageSource>`.
+
+#### Directory.Packages.props
+```xml
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <ItemGroup>
+        <PackageSource key="NuGet.org" Feed="https://nuget.org/api/v3/" />
```

Comment:
That would be the PERFECT place and opportunity to do that :
`<PackageSource key="NuGet.org" Feed="https://nuget.org/api/v3/" Proxy="http://corpo:8080" ProxyAuthentication="NTLMv2|windowscred|networkcredential......" />`